lldb/tools/lldb-dap/package.json

```diff
@@ -244,6 +244,20 @@
         }
       }
     ],
+    "commands": [
+      {
+        "command": "lldb-dap.modules.copyProperty",
+        "title": "Copy Value"
+      }
+    ],
+    "menus": {
+      "view/item/context": [
```

Comment:
Can we also disable the `Copy Value` command from the command prompt?

adding:
```
"commandPalette": [{"command": "lldb-dap.modules.copyProperty", "when": "false"}],
```

Should hide it, since its not useful without the context of a tree view item.